src/components/shared/buttons/SubmitButton/SubmitButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
```

Comment:
Could You please move it from `shared/buttons` directory just to `components` - to keep flat directory structure.